Dockerfile

```diff
@@ -16,6 +16,23 @@ RUN if [ "$DEBUG" = "true" ]; then apt-get update \
 && apt-get install --no-install-recommends -y gdb gdbserver \
 && apt-get clean; fi
 
+WORKDIR "${HOME}"
+RUN mkdir antlr
+WORKDIR "${HOME}"/antlr
+RUN apt-get update && apt-get install --no-install-recommends -y default-jre
+RUN curl -O https://s3.amazonaws.com/artifacts.opencypher.org/M23/Cypher.g4
+RUN curl -O https://www.antlr.org/download/antlr-4.13.2-complete.jar
+RUN java -jar antlr-4.13.2-complete.jar -Dlanguage=Cpp -visitor Cypher.g4
```

Comment:
Can we move this entire process of downloading, installing, running, and removing the JVM to pre-requisites image? I do not see a point of doing that here.